src/pyatsimagebuilder/builder.py

```diff
@@ -441,6 +437,18 @@ def _build_image(self, no_cache=False):
             if 'aux' in line and 'ID' in line['aux']:
                 self.image.id = line['aux']['ID']
 
+            # Log stream from build
+            if 'stream' in line:
+                contents = line['stream'].rstrip()
+                if contents:
+                    self._logger.debug(contents)
+
+                # retrive image ID in steam log
+                if not self.image.id:
```

Comment:
Alternatively, instead of only matching output when no ID was found, if output is always matched then the ID will be the *last* matched log line, which should always be the docker success log line.
```suggestion

```